packages/cli/src/build/helpers/transpile.ts

```diff
@@ -13,12 +13,12 @@ export const transformImports = (target: Target, options: MitosisConfig) => (cod
       // we start by replacing all `context.lite` imports with `context`
       // This Context replace is only needed for non-mitosis components, i.e. plain `.js`/`.ts` files.
       // Mitosis components have logic that transform context import paths correctly.
-      /\.context\.lite(['"][;\)])/g,
+      /\.context\.lite/g,
```

Comment:
We still need to make sure it's the end of a string, i.e. ends with `'` or `"`, otherwise we might replace every instance of `context.lite` in the code, which could be non-imports too

```suggestion
      /\.context\.lite['"]/g,
```